packages/app-desktop/gui/PasswordInput/style.scss

```
@@ -1,3 +1,4 @@
 @use "styles/password-input.scss";
 @use "styles/labelled-password-input.scss";
-@use "styles/password-status-icon.scss";
\ No newline at end of file
+@use "styles/password-status-icon.scss";
```

Stylelint (17.4.0)

[error] Unexpected extension ".scss" in `@use` (scss/load-partial-extension)

(scss/load-partial-extension)